### ISSUE TITLE
please add libbytesize and bcache-tools

### ISSUE THREAD

**Issue by KarstenHopp** [fedora-modularity/baseruntime-package-lists#38]
we have libblockdev-mdraid in platform, but not libbytesize:   'conflicting requests - nothing provides libbytesize.so.1()(64bit) needed by libblockdev-mdraid-2.11-1.module_04d473e8.x86_64'

likewise:'nothing provides bcache-tools >= 1.0.8 needed by libblockdev-kbd-2.11-1.module_04d473e8.x86_64'

**Comment by sgallagher** [comment]
Looks like that's because libblockdev-mdraid is itself not strictly required by anything else in platform, so our dependency solver didn't pick up what it needs to run.
@contyk Do we want to resolve this in platform or filter out libblockdev-mdraid and have them carry it in the udisks2 module? Or do we want to merge the udisks2 module into platform?

**Comment by sgallagher** [comment]
This is also reported as https://github.com/fedora-modularity/hp/issues/22

**Comment by igor-raits** [comment]
@sgallagher I really think that udisks2 should be part of platform.

**Comment by sgallagher** [comment]
@contyk I tend to agree with @ignatenkobrain above; should we move udisks2 into platform?
